pkg/target/cluster.go

```diff
@@ -80,6 +101,55 @@ func (cl *Cluster) Send(cwg *sync.WaitGroup, finish chan struct{}) {
 	}()
 }
 
+func (cl *Cluster) removeAvailableHost(host *Host) {
+	for i, h := range cl.AvailableHosts {
+		if h.Name == host.Name {
```

Comment:
Host comparison should include Port too.
Not sure what is "go way" of doing this. Maybe add an "Equal" method to the Host?

Reply:
Updated the comparison to compare hosts.